zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Publish.java

```diff
@@ -67,6 +87,74 @@ public boolean publish(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    /**
+     * Manual Publish the files in a collection
+     *
+     * @param collection       the collection to publish
+     * @param session          a session with editor priviledges
+     * @param skipVerification
+     * @return success
+     * @throws IOException
+     * @throws UnauthorizedException
+     * @throws BadRequestException
+     * @throws ConflictException     - If there
+     */
+    public boolean publish(Collection collection,
+                           Session session,
+                           boolean breakBeforePublish,
+                           boolean skipVerification)
+            throws IOException, UnauthorizedException, BadRequestException,
+            ConflictException, NotFoundException {
+
+        // Collection exists
+        if (collection == null) {
+            throw new BadRequestException("Please provide a valid collection.");
+        }
+
+        // User has permission
+        if (session == null || !zebedeeCmsService.getPermissions().canEdit(session.getEmail())) {
+            throw new UnauthorizedException(getUnauthorizedMessage(session));
+        }
+
+        // Check approval status
+        if (collection.description.approvalStatus != ApprovalStatus.COMPLETE) {
```

Comment:
if its there can you use `collection.getDescription()` instead? I am trying to phase out the public class memebers when and where I can.

Reply:
Good point. I had just moved this function from the collections class and not considered the contents too much. 👍